pkgs/cronet_http/lib/src/cronet_client.dart

```diff
@@ -333,14 +330,10 @@ class CronetClient extends BaseClient {
     headers.forEach((k, v) => builder.addHeader(k.toJString(), v.toJString()));
 
     if (body.isNotEmpty) {
-      // TODO: Use a more efficient approach when
-      // https://github.com/dart-lang/jnigen/issues/387 is fixed.
-      final bodyBytes = JArray(jbyte.type, body.length);
-      for (var i = 0; i < body.length; ++i) {
-        bodyBytes[i] = body[i];
-      }
+      final bodyBuffer =
+          body.toJByteBuffer().castTo(const jb.$ByteBufferType());
```

Comment:
Actually if you don't generate `java.nio.ByteBuffer` (remove it from `jnigen.yaml`), it will use `package:jni`'s `JByteBuffer` so this cast won't be necessary.

Reply:
`suspend_fun_to_async` is also no longer necessary in `jnigen.yaml`.

Reply:
Done